swarm/api/http/error.go

```diff
@@ -110,7 +110,8 @@ func ShowMultipleChoices(w http.ResponseWriter, r *http.Request, list api.Manife
 //(and return the correct HTTP status code)
 func ShowError(w http.ResponseWriter, r *http.Request, msg string, code int) {
 	if code == http.StatusInternalServerError {
-		log.Error(msg)
+		//log.Error(msg)
+		log.Output(msg, log.LvlError, 3)
```

Comment:
Adding `log.Output` so that we have a correct `calldepth` (3 instead of 2) and get the actual line number and filename of the caller of `ShowError`.

Also `ShowError` abstraction is weird, if not wrong, because this method not only logs an error, but also responds with a number of response codes. This will be refactored next, but at least now we get more meaningful log messages.